redash/tasks/queries.py

```diff
@@ -380,11 +390,11 @@ def _load_data_source(self):
 # user_id is added last as a keyword argument for backward compatability -- to support executing previously submitted
 # jobs before the upgrade to this version.
 @celery.task(name="redash.tasks.execute_query", bind=True, track_started=True)
-def execute_query(self, query, data_source_id, metadata, user_id=None,
+def execute_query(self, query, data_source_id, metadata, user_id=None, is_api_key=False,
```

Comment:
Need to make sure that this is backward compatible with old messages that don't have `is_api_key` (i.e. is Celery really passing those as keyword args, so the order won't matter).

Also it probably makes sense to convert this thing into a dictionary so we don't have to update the sig everytime, but this is for a future refactor.

Reply:
It actually doesn't 👎 
Is 16a8a404 a sufficient remedy?